src/course-unit/hooks.jsx

```diff
@@ -37,7 +37,7 @@ export const useCourseUnit = ({ courseId, blockId }) => {
   const [hasInternetConnectionError, setInternetConnectionError] = useState(false);
   const courseUnit = useSelector(getCourseUnitData);
   const savingStatus = useSelector(getSavingStatus);
-  const loadingStatus = useSelector(getLoadingStatus);
+  const isLoading = useSelector(getLoadingStatus);
```

Comment:
Let's rename `getLoadingStatus` to `getIsLoading` if the selector return boolean value.

Reply:
Changed
Good catch!